dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -221,8 +224,10 @@ def Main():
     print('{0!s}'.format(exception))
     return False
 
-  if not tool.ParseArguments(sys.argv[1:]):
-    # TODO: print errors if this fails.
+  try:
+    tool.ParseArguments(sys.argv[1:])
+  except errors.CommandLineParseError as exception:
+    sys.stderr.write(str(exception))
```

Comment:
Isn't sys.stderr.write() expecting bytes (on Python 3)?

Reply:
Apparently not, write seems to expect `str`

```
$ python3
Python 3.7.3 (default, Mar 27 2019, 09:23:15)
[Clang 10.0.1 (clang-1001.0.46.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.stderr.write(b'asd')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: write() argument must be str, not bytes
```